### ISSUE TITLE
Some Questions about training

### ISSUE THREAD

**Issue by KevinWalkerFan** [HiKapok/SSD.TensorFlow#14]
1.What does the acc(post_forward/cls_accuracy_1) mean? Is it possible to show the AP/mAP during training? 
2. I try half of the batchsize with GTX1060 and try to lower the learning rate and run more steps as u told. Is it also necessary to change the end_learning_rate/decay_boundaries/lr_deacay_factors?

**Comment by HiKapok** [comment]
@KevinWalkerFan you'd also try to change those params if you know how to do. cls_acc is the classification accuracy of all selected postitive and negative anchors. It'ok to show mAP during tringing, but the training speed will be more slower and I did wrote codes for that.

**Comment by KevinWalkerFan** [comment]
Thank you so much for ur instant reply.
